distributed/worker_state_machine.py

```diff
@@ -746,6 +748,30 @@ def _after_from_dict(self) -> None:
         self.value = None
         self.type = None
 
+    @staticmethod
+    def dummy(
+        *,
+        key: str,
+        value: object = None,
+        start: float = 0.0,
+        stop: float = 1.0,
+        nbytes: int = 8,
+        type_: Type | None = None,
```

Comment:
```suggestion
        type: Type | None = None,
```
I think ideally, `dummy` would have the same signature as the non-dummy `__init__`.

Reply:
Done, this was a remnant from `mypy` complaining about `type` being used as a type for `type`. I double-checked and switching the variable name is not a problem.